homeassistant/components/systemmonitor/sensor.py

```diff
@@ -463,7 +463,7 @@ async def async_setup_entry(
 
         sensor_registry[(_type, "")] = SensorData("", None, None, None, None)
         is_enabled = check_legacy_resource(f"{_type}_", legacy_resources)
-        loaded_resources.add(f"{_type}_")
+        loaded_resources.add(slugify(f"{_type}_"))
```

Comment:
Is this needed for `_type` as it is used as a key for `SENSOR_TYPES` and should always match the sensor description key?

Reply:
If all types are already slugs we don't need it, I think. But otherwise we do need it to allow the comparison.

Reply:
I did it like this to make it easy and comparable all the way for types are already slugs so it's the arguments really which needs to be alugified.